src/core/metadata/qgslayermetadata.cpp

```diff
@@ -158,6 +158,11 @@ void QgsLayerMetadata::addKeywords( const QString &vocabulary, const QStringList
   mKeywords.insert( vocabulary, keywords );
 }
 
+bool QgsLayerMetadata::removeKeywords( const QString &vocabulary )
+{
+  return bool( mKeywords.remove( vocabulary ) == 1 );
```

Comment:
That's where I see my future I wondering if that *really* always returns one and if there's a good reason to return false if that isn't the case in a certain scenario for whatever reason.